src/blocks/testimonials/inspector.js

```diff
@@ -0,0 +1,213 @@
+
+const { getComputedStyle } = window;
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { backgroundColor, textColor } = ownProps.attributes;
+	const editableNode = node.querySelector( '[contenteditable="true"]' );
+	//verify if editableNode is available, before using getComputedStyle.
+	const computedStyles = editableNode ? getComputedStyle( editableNode ) : null;
+	return {
+		fallbackBackgroundColor: backgroundColor || ! computedStyles ? undefined : computedStyles.backgroundColor,
+		fallbackTextColor: textColor || ! computedStyles ? undefined : computedStyles.color,
+	};
+} );
+
+const Inspector = ( props ) => {
+	const {
+		activeStyle,
+		attributes,
+		backgroundColor,
+		bubbleBackgroundColor,
+		bubbleTextColor,
+		layoutOptions,
+		onSetColumns,
+		onSetGutter,
+		onToggleImages,
+		onToggleRoles,
+		onUpdateStyleName,
+		setBackgroundColor,
+		setBubbleBackgroundColor,
+		setBubbleTextColor,
+		setTextColor,
+		textColor,
+	} = props;
+
+	const {
+		showImages,
+		showRoles,
+	} = attributes;
+
+	const gutterOptions = [
+		{
+			/* translators: abbreviation for small size */
+			label: __( 'S', 'coblocks' ),
+			tooltip: __( 'Small', 'coblocks' ),
+			value: 'small',
+		},
+		{
+			/* translators: abbreviation for medium size */
+			label: __( 'M', 'coblocks' ),
+			tooltip: __( 'Medium', 'coblocks' ),
+			value: 'medium',
+		},
+		{
+			/* translators: abbreviation for large size */
+			label: __( 'L', 'coblocks' ),
+			tooltip: __( 'Large', 'coblocks' ),
+			value: 'large',
+		},
+		{
+			/* translators: abbreviation for largest size */
+			label: __( 'XL', 'coblocks' ),
+			tooltip: __( 'Huge', 'coblocks' ),
+			value: 'huge',
+		},
+	];
+
+	return (
+		<InspectorControls>
+			<PanelBody
+				initialOpen={ false }
+				title={ __( 'Styles', 'coblocks' ) }>
+				<div className="block-editor-block-styles block-editor-block-styles coblocks-editor-block-styles">
+					{ layoutOptions.map( ( style ) => (
+						<div
+							aria-label={ style.label || style.name }
+							className={ classnames(
+								'block-editor-block-styles__item block-editor-block-styles__item',
+								{
+									'is-active': activeStyle === style.name,
+								}
+							) }
+							key={ `style-${ style.name }` }
+							onClick={ () => onUpdateStyleName( style ) }
+							onKeyDown={ ( event ) => {
+								if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
+									event.preventDefault();
+									onUpdateStyleName( style );
+								}
+							} }
+							role="button"
+							tabIndex="0"
+						>
+							{ <img alt="" src={ `/wp-content/plugins/coblocks/dist/images/testimonial-style-${ style.name }${ ! showImages ? '--noimage' : '' }.png` } /> }
+							<div className="block-editor-block-styles__item-label block-editor-block-styles__item-label">
+								{ style.label || style.name }
+							</div>
+						</div>
+					) ) }
+				</div>
+			</PanelBody>
+
+			<PanelBody
```

Comment:
We have quite a few `PanelBody` elements as well as the `PanelColorSettings` which give us a pan-hell that is not good UX. Can we at minimum combine the color settings into one panel? We should be able to conditionally show them within the panel. 

![image](https://user-images.githubusercontent.com/30462574/152063815-d09cd269-dab2-44c8-b0bf-79121bfda41e.png)